boilerplates/fe-vue/Jenkinsfile

```diff
@@ -0,0 +1,54 @@
+def final projectId = '@project_id@'
+def final componentId = '@component_id@'
+def final credentialsId = "${projectId}-cd-cd-user-with-password"
+def sharedLibraryRepository
+def dockerRegistry
+node {
+  sharedLibraryRepository = env.SHARED_LIBRARY_REPOSITORY
+  dockerRegistry = env.DOCKER_REGISTRY
+}
+
+library identifier: 'ods-library@production', retriever: modernSCM(
+  [$class: 'GitSCMSource',
+   remote: sharedLibraryRepository,
+   credentialsId: credentialsId])
+
+// See readme of shared library for usage and customization.
+odsPipeline(
+  image: "${dockerRegistry}/cd/jenkins-slave-nodejs8-angular",
```

Comment:
There is a 10 now, see #216 - can we use that?

Reply:
sure, why not :-)
Beside the underlying nodeJS version there shouldn't be any interaction between the quickstarter and whatever else is baked into the slave (e.g. Angular CLI or cypress).

Reply:
mhhh, it seems like there actually is a dependency to cypress via `@vue/cli-plugin-e2e-cypress` in devDependencies. This might not work in environments that block post-installation, e.g. during `npm install` with cypress. A workaround would be using the cypress inside the slave directly. Maybe we should put this into the `Known limitations` section of the `README.md`